nx-dev/nx-dev/tailwind.config.js

```diff
@@ -11,13 +11,11 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        black: '#222222',
```

Comment:
Can we use `hsla` values for consistency?

Reply:
Sure. What was the reason for using HSL (and additionally ALPHA, since we don't seem to use it)?